app/frontend/src/components/RequestForm.vue

```diff
@@ -0,0 +1,199 @@
+            solo
+            dense
+            single-line
+            outlined
+            flat
+          >
+            <template v-slot:append-outer></template>
+          </v-text-field>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col>
+          <label>Comments (optional)</label>
+          <v-textarea
+            v-model="comments"
+            rows="3"
+            auto-grow
+            hide-details="auto"
+            dense
+            solo
+            outlined
+            flat
+          ></v-textarea>
+        </v-col>
+      </v-row>
+    </v-form>
+    <div class="justify-center pb-8">
+      <v-btn class="BC-Gov-PrimaryButton light float-left" text @click="cancel()">Cancel</v-btn>
+      <v-btn class="BC-Gov-PrimaryButton float-right" text @click="postRegistrationForm()">Submit</v-btn>
+    </div>
+
+    <BaseDialog v-bind:show="errorOccurred" @close-dialog="errorOccurred = false">
+      <template v-slot:icon>
+        <v-icon large color="red">cancel</v-icon>
+      </template>
+      <template v-slot:text>
+        <p>
+          An error occurred while attempting to add your application.
+          <br />You can also add your application by sending an email to
+          <a
+            href="mailto:NR.CommonServiceShowcase@gov.bc.ca?subject=GETOK Registration for <acronym> - <idir>"
+          >NR.CommonServiceShowcase@gov.bc.ca</a>
+        </p>
+        <p>Please include your Acronym as well as your IDIR username in your email.</p>
+      </template>
+    </BaseDialog>
+
+    <BaseDialog v-bind:show="registerSuccess" @close-dialog="registerSuccess = false">
+      <template v-slot:icon>
+        <v-icon large color="green">check_circle_outline</v-icon>
+      </template>
+      <template v-slot:text>
+        <p>Your Registration was sent successfully. You will be sent an email to {{ userInfo.emailAddress }} when your application has been authorized.</p>
+      </template>
+    </BaseDialog>
+  </v-container>
+</template>
+
+<script>
+import Vue from 'vue';
+import { FieldValidations } from '@/utils/constants.js';
+
+export default {
+  name: 'RequestForm',
+  data: function() {
+
+    return {
+      applicationAcronym: '',
+      applicationAcronymRules: [
+        v => !!v || 'Acronym is required',
+        v =>
+          v.length <= FieldValidations.ACRONYM_MAX_LENGTH ||
+          `Acronym must be ${FieldValidations.ACRONYM_MAX_LENGTH} characters or less`,
+        v =>
+          /^(?:[A-Z]{2,}[_]?)+[A-Z]{1,}$/g.test(v) ||
+          'Incorrect format. Hover over ? for details.'
+      ],
+      comments: '',
+      errorOccurred: false,
+      fieldValidations: FieldValidations,
+      registrationDialog: false,
+      registerSuccess: false,
+      valid: false,
+      userInfo: {
+        emailAddress: Vue.prototype.$keycloak.tokenParsed.email,
+        idir:  Vue.prototype.$keycloak.tokenParsed.preferred_username,
+        name:  Vue.prototype.$keycloak.tokenParsed.name
```

Comment:
Extra space between : and Vue

Optional: Also may want to consider creating a "form" top level object and having all form field content reside under that object instead. It makes it easier to see at a glance what pieces of data are form content, and what other pieces are for visual state.